packages/manager/apps/catalog/vite.config.mjs

```diff
@@ -1,16 +1,17 @@
 import { defineConfig } from 'vite';
 import { getBaseConfig } from '@ovh-ux/manager-vite-config';
 
-export default defineConfig(({ command, mode, ssrBuild }) => {
-    if (command === 'serve') {
-      return {
-        ...getBaseConfig(),
-        base: '/app/'
-      }
-    } else {
-      return {
-        ...getBaseConfig(),
-        base: './'
-      }
-    }
-})
\ No newline at end of file
+export default defineConfig(({ command }) => {
+  const baseConfig = getBaseConfig();
+
+  if (command === 'serve') {
+    return {
+      ...baseConfig,
+      base: '/app/',
+    };
+  }
+  return {
+    ...baseConfig,
+    base: './',
+  };
```

Comment:
Is this part usefull ?
We already have a logic to detect base in the vite-config package.

Reply:
Can you stop resolving comment and answer them please ?

Reply:
It has been delete 👍